framework/source/class/qx/ui/table/columnmenu/MenuItem.js

```diff
@@ -29,12 +29,12 @@ qx.Class.define("qx.ui.table.columnmenu.MenuItem",
     /**
      * Whether the table column associated with this menu item is visible.
      */
-    visible :
+    columnVisible :
     {
       check : "Boolean",
       init  : true,
-      apply : "_applyVisible",
-      event : "changeVisible"
+      apply : "_applyColumnVisible",
+      event : "changeColumnVisible"
     }
```

Comment:
Yes (this API change bit me too, see e948834).
But I can't see how I can stop the property system from creating that "isVisible" method.
My first thought was: "implement `setVisible`, `getVisible` methods and `"changeVisible"` event and remove the property", but the Interface `qx.ui.table.IColumnMenuItem` likes to have a property named `visible`!
Or is it O.K. to "manually" create the methods and event to fulfill the interface's needs?